lib/resty/session/utils.lua

```diff
@@ -592,6 +592,11 @@ local derive_pbkdf2_hmac_sha256 do
     PBKDF2_SHA256_OPTS.salt = usage .. ":" .. salt
     PBKDF2_SHA256_OPTS.pass = pass
 
+    -- Disables the SP800-132 compliance checks on FIPS-mode. This checks has constraints for the iteration count is at least 1000.
+    if is_fips_mode() then
+      PBKDF2_SHA256_OPTS.pkcs5 = 1
+    end
```

Comment:
```suggestion
    -- Disables the SP800-132 compliance checks on FIPS-mode. This checks has constraints for the iteration count is at least 1000.
    PBKDF2_SHA256_OPTS.pkcs5 = 1
```

PBKDF2 is only used in FIPS mode, and we can turn this always on.

Reply:
fixed